app/templates/foundry.html

```diff
@@ -34,10 +33,10 @@ <h1 class="page-title">ADI Foundry</h1>
                     <h4>Apply</h4>
                     <span class="bar gray"></span>
                 </div>
-                <h3>Applications are due by October 2nd, so apply soon!</h3>
-                <div class="button-group">
+                <h3>Applications have not opened yet. Sign up for the ADI newsletter to be notified when applications open!</h3>
+                <!-- <div class="button-group">
                     <div class="button gray"><a target="_blank" href="http://bit.ly/adi-foundry-app">Apply Now</a></div>
```

Comment:
Do we want to just delete this instead of commenting it out? It's possible some enterprising student finds the `href` hidden here.